### ISSUE TITLE
“go install” instructions don’t work

### ISSUE THREAD

**Issue by ticky** [buildkite/terminal#37]
```
$ go install github.com/buildkite/terminal/cmd/terminal-to-html
can't load package: package github.com/buildkite/terminal/cmd/terminal-to-html: cannot find package "github.com/buildkite/terminal/cmd/terminal-to-html" in any of:
        /usr/local/Cellar/go/1.7.5/libexec/src/github.com/buildkite/terminal/cmd/terminal-to-html (from $GOROOT)
        /Users/ticky/go/src/github.com/buildkite/terminal/cmd/terminal-to-html (from $GOPATH)
```

**Comment by mipearson** [comment]
Looking at the readme, this has already been fixed - `go get` works.

**Comment by ticky** [comment]
Oh, yep, fixed in bff759e6fbce9c24f211498c1c0bcc1d177b20b4!
